spec/features/18f_procurement_spec.rb

```diff
@@ -27,7 +27,7 @@
       expect(Dispatcher).to receive(:deliver_new_proposal_emails)
 
       visit '/gsa18f/procurements/new'
-      fill_in "Product Name and Description", with: "buying stuff"
+      fill_in "Product name and description", with: "buying stuff"
```

Comment:
I mean changing these lines to

```ruby
fill_in 'gsa_18f_procurement[product_name_and_description]', with: "buying stuff"
```

The downside there is that you're relying on generated `name` attributes, which seem like they should be more of an implementation detail. How would you do it?